src/pybind/mgr/cephadm/services/monitoring.py

```diff
@@ -119,7 +119,7 @@ def generate_config(self, daemon_spec: CephadmDaemonSpec[AlertManagerSpec]) -> T
                 continue
             if dd.daemon_id == self.mgr.get_mgr_id():
                 continue
-            addr = self.mgr.inventory.get_addr(dd.hostname)
+            addr = self._inventory_get_addr(dd.hostname)
```

Comment:
why are we not modifying https://github.com/ceph/ceph/blob/octopus/src/pybind/mgr/cephadm/services/cephadmservice.py#L125 to add `return socket.getfqdn(addr)` like in the pacific and master versions of this change?